sdk/spec/ilty/itly_spec.rb

```diff
@@ -216,6 +216,48 @@
             'options: {"fake_plugin0"=>#<FakeCallOptions: for plugin 0>})',
           generate_validation_error: true, expect_to_call_action: false, expect_exception: true
       end
+
+      context 'options.validation = DISABLED with on_validation_error' do
+        include_examples 'validate and run on plugins',
+          method: :identify, pass_data_as: :properties,
+          method_params: { user_id: '123',
+                           properties: { data: '1', info: 'yes' },
+                           options: { 'fake_plugin0' => FakeCallOptions.new(data: 'for plugin 0') } },
+          validation_value: Itly::Options::Validation::DISABLED,
+          set_on_validation_error: true,
+          expected_event_properties: { data: '1', info: 'yes' },
+          expected_log_info: 'identify(user_id: 123, properties: {:data=>"1", :info=>"yes"}, '\
+            'options: {"fake_plugin0"=>#<FakeCallOptions: for plugin 0>})',
+          generate_validation_error: true, expect_validation: false
```

Comment:
Does this mean `on_validation_error` will be called in the `Validation::DISABLED` case?

Reply:
It means `on_validation_call` is not expected in the Validation::DISABLED case.
 
https://github.com/amplitude/itly-sdk-ruby/blob/DXOC-591-on-validation-error/sdk/spec/support/shared_examples/validate_and_run_on_plugins.rb#L97-L104